udata/api/__init__.py

```diff
@@ -145,7 +148,7 @@ def page_parser(self):
 
 
 @api.representation('application/json')
-def output_json(data, code, headers=None):
+def output_json(data, code=httplib.OK, headers=None):
```

Comment:
You need to keep the same signature as [Flask's `make_response`](http://flask.pocoo.org/docs/0.12/api/#flask.make_response) which accepts a unique parameter. I just noticed that it should allow an empty call but in our case it doesn't make sense.